draft-lmbdhk-quic-multipath.md

```diff
@@ -256,6 +256,71 @@ Senders MUST manage per-path congestion status, and MUST NOT send more data on a
 
 Multipath TCP uses the LIA congestion control scheme specified in {{RFC6356}}.  This scheme can immediately be adapted to Multipath QUIC. Other coupled congestion control schemes have been proposed for Multipath TCP such as {{OLIA}}.
 
+# Path States
+
+{{fig-path-states}} shows the states that an endpoint's path can have.
+
+~~~
+       o
+       | Initiate path validation
+       | PATH_CHALLENGE received on new path
```

Comment:
Isn't the first step to send (not received) a PATH_CHALLENGE and then you stay in "Validating" state until you receive PATH_RESPONSE (which would trigger the transition to the next state)?

Reply:
I think I would omit the part in this diagram where  PATH_CHALLENGE is received by not not send and just define that the validating is enter as soon as a PATH_CHALENGE is sent. Or if you want to show this in the diagram then we would need a separate state for this? Or maybe a separate diagram for the initiator and the "responder"?